conans/test/functional/toolchains/cmake/test_ninja.py

```diff
@@ -78,7 +78,7 @@ def test_locally_build_linux(build_type, shared, client):
     assert 'cmake -G "Ninja"' in client.out
     assert "main: {}!".format(build_type) in client.out
     client.run("install hello/1.0@ -g=deploy -if=mydeploy {}".format(settings))
-    ldpath = os.path.join(client.current_folder, "mydeploy", "hello")
+    ldpath = os.path.join(client.current_folder, "mydeploy", "hello", "lib")
```

Comment:
Why? This seems to be fine without this change, and also with this change?

Reply:
I've added `LIBRARY DESTINATION lib`, so libraries are deployed to the `lib` sub-directory with `gen_cmakelists`.
otherwise, all recipes will have to add `self.package_info.libdirs.append(self.package_folder)` (while `lib` is automatically added by conan)